versioning/invoke_embedme.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Python version 3.4 or higher is required to run this script.
+
+
+
+import argparse
+import os
+import sys
+import urllib.request
+
+# NPX command.
+npx_command = 'npx embedme'
+
+def delete_from_file(the_file):
+    if os.path.exists(the_file):
+      os.remove(the_file)
+
+def main():
+    # load file from azure-sdk-for-java
+    keyVaultJcaManagedIdentitySample_path = os.path.abspath("../keyvault/azure-spring-boot-sample-keyvault-certificates-client-side/src/main/java/com/azure/spring/security/keyvault/KeyVaultJcaManagedIdentitySample.java")
```

Comment:
This is used for test?

Reply:
not for test

Reply:
So we only check one file? My expectation is to check all files.